cmd/admin-handlers.go

```diff
@@ -478,6 +483,90 @@ func (a adminAPIHandlers) GetConfigHandler(w http.ResponseWriter, r *http.Reques
 	writeSuccessResponseJSON(w, econfigData)
 }
 
+// Disable tidwall json array notation in JSON key path so
+// users can set json with a key as a number.
+// In tidwall json, notify.webhook.0 = val means { "notify" : { "webhook" : [val] }}
+// In Minio, notify.webhook.0 = val means { "notify" : { "webhook" : {"0" : val}}}
+func normalizeJSONKey(input string) (key string) {
+	subKeys := strings.Split(input, ".")
+	for i, k := range subKeys {
+		if i > 0 {
+			key += "."
+		}
+		if _, err := strconv.Atoi(k); err == nil {
```

Comment:
Should this always be a number? Can it not be a string?

Reply:
> Should this always be a number? Can it not be a string?

normalizeJSONKey() transforms, as an example, `notify.webhook.0` to `notify.webhook.:0` because tidwall only understands the former format. So I am only transforming numbers, there is nothing to do with strings

Reply:
ok